runtime/shiden/src/lib.rs

```diff
@@ -1259,16 +1264,53 @@ pub type Executive = frame_executive::Executive<
     Migrations,
 >;
 
+pub struct StaticTierParamsMigration;
+impl OnRuntimeUpgrade for StaticTierParamsMigration {
+    fn on_runtime_upgrade() -> Weight {
+        let onchain_storage_version =
+            pallet_dapp_staking_v3::Pallet::<Runtime>::on_chain_storage_version();
+
+        if onchain_storage_version == 8 {
+            let tier_thresholds = BoundedVec::try_from(vec![
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(35_700_000), // 3.57%
+                    minimum_required_percentage: Perbill::from_parts(23_800_000), // 2.38%
+                },
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(8_900_000), // 0.89%
+                    minimum_required_percentage: Perbill::from_parts(6_000_000), // 0.6%
+                },
+                TierThreshold::DynamicPercentage {
+                    percentage: Perbill::from_parts(23_800_000), // 2.38%
+                    minimum_required_percentage: Perbill::from_parts(17_900_000), // 1.79%
+                },
+                TierThreshold::FixedPercentage {
+                    required_percentage: Perbill::from_parts(6_000_000), // 0.6%
+                },
+            ])
+            .unwrap();
```

Comment:
Suggestion - provide this as a configurable type to the pallet_dapp_staking_v3::migration::versioned_migrations::V7ToV8<Runtime>.

That way the entire migration is ensured via the `VersionedMigration` struct.